second-analysis-steps/simulation-intro.md

```diff
@@ -20,3 +20,28 @@ Gauss is the LHCb simulation framework which manages the creation of simulated e
 3. The number of generated events refers to the number entering step 4 above, so those passing the generator level cuts. __Not__ the number of events produced by the `ProductionTool` in the first step.
 
 {% endcallout %}
+
+## Choosing your Gauss Version
+
+Each release of Gauss v49 represents a different release of Sim09. You may want to run an older version of Gauss to verify something in a pre-existing MC dataset or for any number of other reasons. Here is a table matching each release to each Sim09 realease:
```

Comment:
"matching each release to each Sim09 realease" -> suggest changing one of 'release' to 'version'